truss/templates/triton/root/generate_config.py

```diff
@@ -0,0 +1,196 @@
+import importlib
```

Comment:
I think it would be good to have some unit tests (not integration) that take some trusses and checks to see whether the correct triton config is outputted, there's a lot of logic happening here